clients/admin-ui/src/features/common/table/v2/FidesTable.tsx

```diff
@@ -119,6 +121,7 @@ const TableBody = <T,>({
   onRowClick,
   renderRowTooltipLabel,
   displayAllColumns,
+  emptyTableNotice,
```

Comment:
Updated `FidesTable` to accept an `emptyTableNotice` component to display inside the table body. I updated the Properties and Add Vendor tables to use this new design.
<img width="1181" alt="image" src="https://github.com/ethyca/fides/assets/2092272/634e391c-03f2-4c09-be00-ef7b99e2d6e1">

<img width="1199" alt="image" src="https://github.com/ethyca/fides/assets/2092272/36c3376f-2125-4f63-8088-3cdbaf95f263">